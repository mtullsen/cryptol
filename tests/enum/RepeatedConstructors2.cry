enum T = A | X
newtype X = { x : [8] }
