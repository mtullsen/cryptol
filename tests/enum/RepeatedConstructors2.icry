:load RepeatedConstructors2.cry
