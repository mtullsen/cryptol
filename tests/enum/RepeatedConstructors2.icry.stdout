Loading module Cryptol
Loading module Cryptol
Loading module Main

[error]
    Overlapping symbols defined:
    (at RepeatedConstructors2.cry:1:14--1:15, Main::X)
    (at RepeatedConstructors2.cry:2:9--2:10, Main::X)
